appcenter-analytics/ios/AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.m

```diff
@@ -31,43 +37,86 @@ @implementation AppCenterReactNativeAnalytics
 
 + (void)registerWithInitiallyEnabled:(BOOL) enabled
 {
-    [AppCenterReactNativeShared configureAppCenter];
-    [MSAppCenter startService:[MSAnalytics class]];
-    if (!enabled) {
-        [MSAnalytics setEnabled:enabled];
-    }
+  [AppCenterReactNativeShared configureAppCenter];
+  [MSAppCenter startService:[MSAnalytics class]];
+  if (!enabled) {
+    [MSAnalytics setEnabled:enabled];
+  }
+}
+
+- (MSTargetsDictionary *)targetsForTokens
+{
+  if (self.transmissionTargets == nil) {
+    self.transmissionTargets = [MSTargetsDictionary new];
+  }
+  return self.transmissionTargets;
 }
 
 RCT_EXPORT_METHOD(isEnabled:(RCTPromiseResolveBlock)resolve
-                    rejecter:(RCTPromiseRejectBlock)reject)
+                  rejecter:(RCTPromiseRejectBlock)reject)
 {
-    resolve([NSNumber numberWithBool:[MSAnalytics isEnabled]]);
+  resolve([NSNumber numberWithBool:[MSAnalytics isEnabled]]);
 }
 
 RCT_EXPORT_METHOD(setEnabled:(BOOL)shouldEnable
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
-    [MSAnalytics setEnabled:shouldEnable];
-    resolve(nil);
+  [MSAnalytics setEnabled:shouldEnable];
+  resolve(nil);
 }
 
 RCT_EXPORT_METHOD(trackEvent:(NSString *)eventName
-              withProperties:(NSDictionary *)properties
-                    resolver:(RCTPromiseResolveBlock)resolve
-                    rejecter:(RCTPromiseRejectBlock)reject)
+                  withProperties:(NSDictionary *)properties
```

Comment:
Which formatter are you using here?  Our convention (though I don't like it) is to align the `:`s